frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -219,6 +219,9 @@ const ReceiverBaseForm: React.FC<ReceiverBaseFormProps> = ({
   }
 
   const doesReceiverNameAlreadyExist = (receiverName: string): boolean => {
+    if (!config?.receivers) {
+      return false;
+    }
     const receiverNames = config?.receivers
```

Comment:
Optional chaining here is not needed since you check it above

Reply:
thanks, fixed